src/main/webapp/script.js

```diff
@@ -0,0 +1,100 @@
+var zoomedIn = false;
+var aqLayerOn = true;
+var routeVisible = false;
+
+function changeBackgroundImage(imagePath) {
+    document.body.style.backgroundImage = "url('" + imagePath + "')";
+}
+
+function getImagePath() {
+    path = "";
+    if (zoomedIn) {
+        path = "images/wagga_";
```

Comment:
Prefer anchored paths (ie, leading /) to ensure absolute references, especially for paths.